great_expectations/core/batch_manager.py

```diff
@@ -144,10 +142,7 @@ def reset_batch_cache(self) -> None:
         self._batch_cache = OrderedDict()
         self._active_batch_id = None
 
-    def load_batch_list(self, batch_list: Optional[List[Batch]]) -> None:
-        if batch_list is None:
-            batch_list = []
-
+    def load_batch_list(self, batch_list: List[Batch]) -> None:
```

Comment:
Are there are repercussions from removing the default value here?

Reply:
I just had to update two callers to ensure `batch_list` is not `None`. Those are the only two places this is called.